lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -364,42 +377,64 @@ class AutofillInfo {
   /// Used as a guidance to the browser as to the type of information expected
   /// in the field.
   /// See: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete
-  final String hint;
+  final String? autofillHint;
```

Comment:
nit: is the first sentence in the dartdoc still useful? Seems obvious now that this is used for autofill.